modules/configuring-albo-on-sts-cluster.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+// * networking/installing-albo-sts-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="nw-installing-albo-on-sts-cluster_{context}"]
+= Installing AWS Load Balancer Operator on Secure Token Service cluster
+
+The Cloud Credential Operator does not provision the credential request automatically in the STS cluster. You must set the credential request manually. You must set the credentials secret manually by using the `ccoctl` binary. The Cloud Credential Operator waits until the required secrets are created and available.
+
+.Prerequisites
+
+* You must extract and save the required `CredentialsRequest` in a directory. Run the following command:
+
+[source,terminal]
+----
+$ oc get credentialsrequest -n openshift-cloud-credential-operator  \
+    aws-load-balancer-controller-<cr-name> -o yaml > <path-to-credrequests-dir>/cr.yaml
+----
+
+* Use the `ccoctl` tool to process all `CredentialsRequest` objects from the `credrequests` directory. Run the following command:
+
+[source,terminal]
+----
+$ ccoctl aws create-iam-roles \
+    --name <name> --region=<aws_region> \
+    --credentials-requests-dir=<path-to-credrequests-dir> \
+    --identity-provider-arn <oidc-arn>
+----
+
+* Apply the secrets to your cluster. Run the following command:
+
+[source,terminal]
+----
+$ ls manifests/*-credentials.yaml | xargs -I{} oc apply -f {}
+----
+
+* Verify that the `aws-load-balancer-controller` pod is created:
+
+[source,terminal]
+----
+$ oc -n aws-load-balancer-operator get pods
+NAME                                                            READY   STATUS    RESTARTS   AGE
+aws-load-balancer-controller-cluster-9b766d6-gg82c              1/1     Running   0          137m
+aws-load-balancer-operator-controller-manager-b55ff68cc-85jzg   2/2     Running   0          3h26m
+----
```

Comment:
Instead of these steps we need the following steps: [link](https://github.com/openshift/aws-load-balancer-operator/blob/b5453e090a98fc20debad44c476ed953e423627e/docs/prerequisites.md#for-sts-clusters). They should be under `Procedure to boostrap AWS Load Balancer Operator` chapter.

Reply:
acked

Reply:
I created a separate file for bootstrapping ALBO procedure